python/composio/__init__.py

```diff
@@ -2,17 +2,30 @@
 
 
 try:
+    import json
     import sentry_sdk
+    from pathlib import Path
+
+    def _hook():
+        user_file = Path.home() / ".composio" / "user_data.json"
+        if not user_file.exists():
+            return
+
+        try:
+            json.loads(user_file.read_text(encoding="utf-8"))["api_key"]
+        except Exception:
+            return
+
+        sentry_sdk.init(
+            dsn=(
```

Comment:
It's a good practice to avoid hardcoding the DSN for Sentry SDK initialization directly in the codebase. Consider using environment variables or a configuration file to manage sensitive information securely.